src/legacy/ui/public/styles/_legacy/_base.scss

```diff
@@ -59,8 +59,8 @@ input[type="checkbox"],
     padding-bottom: $euiSizeS;
   }
 
-  > kbn-top-nav {
-    z-index: 5;
+  .globalQueryBar {
```

Comment:
This should go in this file: https://github.com/elastic/kibana/blob/master/src/legacy/core_plugins/data/public/filter/filter_bar/_global_filter_group.scss

Not legacy styles.

Reply:
done